docs/Reference/Console/Features/ocr.md

```diff
@@ -0,0 +1,87 @@
+
+- **Local Pre-Boot Application (PBA)**: Launch a locally installed recovery or diagnostic tool.
+
+!!! note "Note - OCR support"
+
+    Console currently supports HTTPS Network Boot over a wired connection. Additional recovery options will be added in future releases.
+
+### Prerequisites for HTTPS Boot
+
+Before using HTTPS Boot OCR, ensure the following prerequisites are met:
+
+1. Only Intel AMT version 15 and later support One Click Recovery feature.
+2. Enable HTTP/HTTPS Boot in BIOS settings. This may be disabled by default on some devices.
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBOOT_BIOS.png" alt="Figure 1: Enable HTTPS Boot in BIOS">
+    </figure>
+3. Set up an HTTPS server to host the ISO. For faster booting, run it on the same network as the device.
+    - This guide references using [simple-webserver](https://github.com/jsastriawan/simple-webserver) for the HTTPS server setup. 
+    - In this example, the HTTPS server is running on the same machine as the Console: https://192.168.88.250:5443/ubuntu.iso.
+    - The hosted ISO is a full Ubuntu LTS image served via simple-webserver.
+
+### Triggering OCR via Console
+
+1. Connect to the Intel AMT device using Console over TLS.
+
+2. Enable `HTTPS Network boot` feature in `General AMT Info` Section.
+   
+    !!! note "Note - HTTPS Network Boot supported?"
+
+        If the HTTPS Network Boot checkbox is greyed out, HTTPS Network Boot is not supported on this platform.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_Enable_HTTPS_BOOT.png" alt="Figure 2: Enable HTTPS Network Boot">
+    </figure>
+
+3. Upload the Root Certificate of the HTTPS server hosting the ISO via the `Add New` certificates option.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_ADD_TRUSTEDROOTCERT.png" alt="Figure 3: Add Root Certificate of HTTPS Server">
+    </figure>
+
+4. Click on the three-dot menu and select **Reset to HTTPS Boot (OCR)**.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_Reset_to_HTTPS_Boot.png" alt="Figure 4: Reset to HTTPS Boot (OCR)">
+    </figure>
+
+5. Enter the ISO URL (e.g., https://192.168.88.250:5443/ubuntu.iso). Optionally, enable `Enforce Secure Boot` to boot only a secure `.iso` file.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBOOT_URL.png" alt="Figure 5: URL to the .iso hosted on HTTPS Server">
+    </figure>
+
+6. Click `OK` to start the recovery process. The device will restart and boot from the ISO.
+
+7. Optionally, Connect to KVM and verify that the device loads the ISO.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBoot_Recovery_Start.png" alt="Figure 7: View KVM screen while the ISO boots">
+    </figure>
+ 
+### Troubleshooting and Known issues
```

Comment:
I'd break these out. Requiring a wired connection is a prerequisite, not a known issue or a troubleshooting step.